### PR TITLE
Read password from stdin when admin resets password

### LINKED ISSUE #14076: grafana-cli admin password reset should get the password from stdin
### What Grafana version are you using?
Grafana 1.14.6
### What did you do?
Tried to reset the admin password from the grafana-cli.

`grafana-cli admin reset-admin-password myeasypassword`

The admin password is being set on a command line [argument](https://github.com/grafana/grafana/blob/b47a4954c9b0208869298c75e97c212430565f1e/pkg/cmd/grafana-cli/commands/reset_password_command.go#L16). This command might be saved on bash history and it can be a security issue. 

We could instead get the password from stdin.

### PR DESCRIPTION
**Pull request by praveensastry**
Fixes #14076

### CODE REVIEW

File: pkg/cmd/grafana-cli/commands/reset_password_command.go [hunk 1]
```diff
@@ -2,18 +2,26 @@ package commands
...
 import (
 	"fmt"
+	"syscall"
 
 	"github.com/fatih/color"
 	"github.com/grafana/grafana/pkg/bus"
 	"github.com/grafana/grafana/pkg/cmd/grafana-cli/logger"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/util"
+	"golang.org/x/crypto/ssh/terminal"
```
Comment (bergquist) [inline pkg/cmd/grafana-cli/commands/reset_password_command.go]: you need to add this to the vendor folder with dep.

`dep ensure -add golang.org/x/crypto/ssh/terminal`

File: pkg/cmd/grafana-cli/commands/reset_password_command.go [hunk 2]
```diff
@@ -2,18 +2,26 @@ package commands
...
 
 func resetPasswordCommand(c CommandLine) error {
-	newPassword := c.Args().First()
+
+	bytePassword, err := terminal.ReadPassword(int(syscall.Stdin))
+	if err != nil {
+		fmt.Printf("Failed to read password: %v", err)
+	}
+	newPassword = string(bytePassword)
+	fmt.Println()
```
Comment (bergquist) [inline pkg/cmd/grafana-cli/commands/reset_password_command.go]: I guess this Println should be removed? :)

Comment (praveensastry) [reply pkg/cmd/grafana-cli/commands/reset_password_command.go]: I believe it's necessary to add this new line, please refer [here](https://go.googlesource.com/crypto/+/master/ssh/terminal/util.go)

**Comment by praveensastry** [comment]
Pushed a wrong commit by mistake, closing this one and will open a new one which is much cleaner. sorry for the inconvenience

**Comment by marefr** [comment]
Okay. Just a pointer for the future, you can use git to rewrite commits and force push your branch.

**Pull request closed without merging**